tools/tf2openapi/cmd/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"github.com/golang/protobuf/proto"
+	"github.com/kfserving/tools/tf2openapi/generator"
+	pb "github.com/tensorflow/tensorflow/tensorflow/go/core/protobuf"
+	"io/ioutil"
+	"log"
+)
+
+var model = flag.String("model", "", "Absolute path of SavedModel file")
+
+func main() {
+	flag.Parse()
+
+	in, err := ioutil.ReadFile(*model)
+	if err != nil {
+		log.Print(*model)
+		log.Fatalln("Error reading file:", err)
```

Comment:
I like log.Fatalln because the user has likely supplied a non-existent file, so it would be good to log and terminate instead of printing a stack trace as well with panic. (esp since it's a user error and not a programmer error)